core/model/patron.py

```diff
@@ -249,6 +249,14 @@ def loan_activity_max_age(self):
         """
         return 15 * 60
 
+    def has_last_loan_activity_expired(self):
```

Comment:
I think this might be more appropriately named `is_last_loan_activity_stale`, given that it might never have been set. Feel free to come up with a better name here.